ruby/rust-toolchain

```diff
@@ -0,0 +1,6 @@
+# This file specifies the Rust version used to develop and test the Ruby
+# extractor. It is set to the lowest version of Rust we want to support.
+
+[toolchain]
```

Comment:
Let's also include this file in the cache-key in the following files https://github.com/github/codeql/pull/7799/commits/6451a71a7845b49b83802b8b2de370fbfed24f30